engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/callcaching/CallCacheInvalidateActor.scala

```diff
@@ -0,0 +1,36 @@
+package cromwell.engine.workflow.lifecycle.execution.callcaching
+
+import akka.actor.{Actor, ActorLogging, Props}
+
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success}
+
+class CallCacheInvalidateActor(callCache: CallCache, cacheId: MetaInfoId) extends Actor with ActorLogging {
+
+  implicit val ec: ExecutionContext = context.dispatcher
+
+  def receiver = context.parent
+
+  callCache.invalidate(cacheId) onComplete {
+    case Success(_) =>
+      receiver ! CallCacheInvalidatedSuccess
+      context.stop(self)
+    case Failure(t) =>
+      receiver ! CallCacheInvalidatedFailure(t)
```

Comment:
it means something went wrong at the DB level when trying to flip the allowResultReuse flag to 0. So the next time a call matches the hash of this call it will still be considered a valid cache hit.
Currently the EJEA doesn't actually do anything with except logging it though.